### DISCUSSION TITLE
Are the Crazyflie 2.0 propellers the same as the 2.1?

### DISCUSSION THREAD

**Discussion by loraine92** [bitcraze/discussions#1028] [Q&A]
I have a crazyflie 2.1 and when I do the propeller test from the console when I try with the crazyflie 2.0 replacement propellers I get that they are failing, I would like to know if they are compatible? And if they are compatible, why do they all go wrong?

**Comment by knmcguire** [comment]
Yes, the crazyflie 2.0 props are the same as crazyflie 2.1 props, or at least should be.

If the prop test is failing, either the propellors are not good or the motors shafts are bent (which is not the case if these normally work on good props). It might be that the plastic of the props that you are having are old and deformed? We upgraded to 2.1 already for 5 years.
